src/Event/ReadModel/RDF/EventJsonToTurtleConverter.php

```diff
@@ -77,6 +77,7 @@ final class EventJsonToTurtleConverter implements JsonToTurtleConverter
     private const TYPE_PERIOD = 'm8g:PeriodOfTime';
     private const TYPE_DATE_TIME = 'xsd:dateTime';
```

Comment:
@Stiksels your comment made me wonder if this `dateTime` type should also have a leading capital?

Reply:
Good question, will wait to merge until we get an answer.

Reply:
Ok, so it's not a class but a data type.